examples/network/other/saveAndLoad.html

```diff
@@ -141,30 +132,53 @@
                 return new vis.DataSet(networkNodes);
             }
 
+            function getNodeById(data, id) {
+                var foundNode;
+
+                data.forEach(function(node) {
```

Comment:
It's a minor thing but in this looping an old fashioned 
for(var i=0 )
has the benefit of being able to 
break;
or even a 
return node;